fv3gfs-physics/fv3gfs/physics/functions/microphysics_funcs.py

```diff
@@ -1,6 +1,7 @@
 from gt4py import gtscript
+from gt4py.gtscript import exp, log, sqrt
 
-from fv3gfs.physics.global_constants import *
+import fv3gfs.physics.global_constants as constants
```

Comment:
On one hand the math looks a little weird with `constants` everywhere, but on the other hand it's really helpful to be able to read the code and know what is a constant and what is a variable.